tools/run_benchmark.sh

```diff
@@ -2,13 +2,15 @@
 
 run_benchmark() {
    # shellcheck disable=SC2086
-   python ./tools/test_full_sync.py run $3 --profile --test-constants "$1" &
+   python ./tools/test_full_sync.py run $3 --profile --node-profiler --test-constants "$1" &
```

Comment:
it just occurred to me that combining `--profile` and `--node-profiler` here will cause two `cProfile` instances to be active at the same time. Is that supported by python? I don't recall anything stating it isn't..